src/app/settings/settings/settings.component.spec.ts

```diff
@@ -91,4 +94,55 @@ describe('SettingsComponent', () => {
       new ActionSettingsChangeAutoNightMode({ autoNightMode: false })
     );
   });
+
+  it('should dispatch change animations page', () => {
+    dispatchSpy = spyOn(store, 'dispatch');
+    const componentDebug = fixture.debugElement;
+    const slider = componentDebug.queryAll(By.directive(MatSlideToggle))[1];
+
+    slider.triggerEventHandler('change', { checked: false });
+    fixture.detectChanges();
+
+    expect(dispatchSpy).toHaveBeenCalledTimes(2);
+    expect(dispatchSpy).toHaveBeenCalledWith(
+      new ActionSettingsChangeAnimationsPage({ pageAnimations: false })
+    );
+  });
+
+  it('should dispatch change animations elements', () => {
+    dispatchSpy = spyOn(store, 'dispatch');
+    const componentDebug = fixture.debugElement;
+    const slider = componentDebug.queryAll(By.directive(MatSlideToggle))[2];
+
+    slider.triggerEventHandler('change', { checked: false });
+    fixture.detectChanges();
+
+    expect(dispatchSpy).toHaveBeenCalledTimes(2);
+    expect(dispatchSpy).toHaveBeenCalledWith(
+      new ActionSettingsChangeAnimationsElements({ elementsAnimations: false })
+    );
+  });
+
+  it('should disable change animations page when disabled is set in state', () => {
+    store.setState({
+      theme: 'DEFAULT-THEME',
+      autoNightMode: true,
+      pageAnimations: true,
+      pageAnimationsDisabled: true, // change animations disabled
+      elementsAnimations: true
+    });
+    fixture.detectChanges();
+
+    dispatchSpy = spyOn(store, 'dispatch');
+    const componentDebug = fixture.debugElement;
+    const slider = componentDebug.queryAll(By.directive(MatSlideToggle))[1];
+
+    slider.triggerEventHandler('change', { checked: false });
+    fixture.detectChanges();
+
+    expect(dispatchSpy).toHaveBeenCalledTimes(0);
+    expect(dispatchSpy).not.toHaveBeenCalledWith(
+      new ActionSettingsChangeAnimationsPage({ pageAnimations: false })
```

Comment:
We do NOT need to check that it was not called with some specific action because we know it has been called 0 times.

As a general rule, the less code to achieve the same the better!

besides that the PR is great, thank you for your contribution!